server/entry.sh

```diff
@@ -1,2 +0,0 @@
-echo "Test script"
```

Comment:
We need to define 1 shell script for CI/CD. Without this we cannot pass CI test on github and merge will fail. Just re-add this file and commit again for a new pull request.